### PR TITLE
updated rubocop config to v0.71.0, set rubocop version in hound

### PR DESCRIPTION
**Pull request by yihyang**
1. Set a version of rubocop through `hound`
  - Refer to: http://help.houndci.com/configuration/linter-versioning
2. Updated `.rubocop.yml`'s configuration according to `v0.71.0` settings

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -177,14 +177,26 @@ Naming/FileName:
   Enabled: false
   Exclude: []
 
-Layout/FirstParameterIndentation:
-  Description: Checks the indentation of the first parameter in a method call.
+Layout/IndentFirstArgument:
+  Description: 'Checks the indentation of the first argument in a method call.'
   Enabled: true
+  VersionAdded: '0.68'
```
Comment (BenR1312) [inline .rubocop.yml]: this version is different then the version in the `.hound.yml` does that matter?

**Review by BenR1312 — APPROVED** [review]
Right so it was just an outdated version of the rubocop yml?
Did hound just try to always use latest version or something?

File: .rubocop.yml (hunk 1 above)
Comment (yihyang) [reply .rubocop.yml]: https://github.com/rubocop-hq/rubocop/issues/5978
it's just a meta data that specify which version was this added
I copied it from https://github.com/rubocop-hq/rubocop/blob/64f6ca77af6aa14dc754f9cc5a8fbf32d4fa4f74/config/default.yml#L669
Let me remove it then

**Comment by yihyang** [comment]
@BenR1312 to be exact I'm not quite sure what causes it to not function (can't find anything from logs), what I have done is basically is to try to lock the rubocop version in hound and update the `rubocop.yml` to the latest version

**Comment by BenR1312** [comment]
👍

**Review by kinmor06 — APPROVED** [review]
finally after 1 year :D

**Comment by yihyang** [comment]
Thanks @BenR1312 @kinmor06 !

**Pull request merged**